README.md

```diff
@@ -99,6 +99,7 @@ What it sets up
 [hub]: https://github.com/github/hub
 [ImageMagick]: http://www.imagemagick.org/
 [MySQL]: https://www.mysql.com/
+[n]: https://github.com/tj/n
```

Comment:
I've only used [nvm](https://github.com/creationix/nvm), but no strong feelings about it.